airflow/providers/amazon/aws/example_dags/example_boto3.py

```diff
@@ -0,0 +1,121 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Can you make it a system test? System tests are basically example dags which can be run using `pytest`. The benefit is double:
- having an example dag to help users use a specific operator/sensor (and provide code snippets for documentation)
- having a test for a specific operator/sensor

You can find example of AWS system tests [here](https://github.com/apache/airflow/tree/main/tests/system/providers/amazon/aws). You also can read the README [here](https://github.com/apache/airflow/tree/main/tests/system/providers/amazon) which can provide you some information on how AWS system tests are written

Reply:
[Renamed example](https://github.com/apache/airflow/pull/27656/commits/abc3fbbb757cb5b1b4bcf238b5c5c186c12b36ad). Is there any guides how to setup test env, so I can run it locally ?